util/oidc/oidc.go

```diff
@@ -103,21 +103,26 @@ func NewClientApp(settings *settings.ArgoCDSettings, dexServerAddr, baseHRef str
 	if err != nil {
 		return nil, fmt.Errorf("parse redirect-uri: %v", err)
 	}
-	tlsConfig := settings.OIDCTLSConfig()
+
+	transport := &http.Transport{
```

Comment:
Yes, because of the types. Once you assign the transport to the client, it becomes a RoundTripper instead of an http.Transport and you can't modify the fields directly.

Reply:
Thanks for explanation